src/7bb60_len_41b0.c

```diff
@@ -266,7 +291,18 @@ void collision_check_player_overlaps(void) {
 
 INCLUDE_ASM(s32, "7bb60_len_41b0", phys_should_player_be_sliding);
 
-INCLUDE_ASM(s32, "7bb60_len_41b0", phys_is_on_sloped_ground);
+s32 phys_is_on_sloped_ground(void) {
+    Shadow* playerShadow = get_shadow_by_index(gPlayerStatus.shadowID);
+    f32 rotZ = playerShadow->rotation.z + 180.0;
+    f32 rotX = playerShadow->rotation.x + 180.0;
+    s32 ret = TRUE;
+
+    if (fabsf(rotZ) < 20.0f && fabsf(rotX) < 20.0f) {
+        ret = FALSE;
+    }
+
+    return ret;
```

Comment:
Gonna cry if `return !(fabsf(rotZ) < 20.0f && fabsf(rotX) < 20.0f);` doesn't match

Reply:
I don't think it does, because it stores 1 into a register really early on which a single return statement didn't do, from what I remember. I can try again though

Reply:
does not work, unfortunately